spec/cucumber/multiline_argument/data_table_spec.rb

```diff
@@ -8,9 +8,9 @@ module MultilineArgument
     describe DataTable do
       before do
         @table = DataTable.from([
-          %w{one four seven},
-          %w{4444 55555 666666}
-        ])
+                                  %w{one four seven},
```

Comment:
I don't feel all that strongly about it, but I personally prefer the original indentation. What do others think?